qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/gsas2/call_G2sc.py

```diff
@@ -90,10 +90,8 @@ def enable_unit_cell(refine, override_unit_cell_lengths, project):
 
 def enable_limits(x_limits, project):
```

Comment:
is there a test for this?

Reply:
As discussed offline (but commenting here for anyone else interested) only the model and presenter have tests